modules/aws-dynamodb/src/main/java/org/testcontainers/dynamodb/DynamoDbContainer.java

```diff
@@ -0,0 +1,82 @@
+package org.testcontainers.dynamodb;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.utility.DockerImageName;
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * Container for official AWS DynamoDB.
+ */
+public class DynamoDbContainer extends GenericContainer<DynamoDbContainer> {
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("amazon/dynamodb-local");
+    private static final String DEFAULT_TAG = "1.13.5";
+    private static final int MAPPED_PORT = 8000;
+
+    /**
+     * @deprecated use {@link DynamoDbContainer (DockerImageName)} instead
+     */
+    @Deprecated
+    public DynamoDbContainer() {
```

Comment:
do we need this constructor?  can it be removed?

Reply:
Nope, we don't. Fixed!